### ISSUE TITLE
Perhaps sealer needs to provide an application image market

### ISSUE THREAD

**Issue by lllwan** [sealerio/sealer#1433]
## Issue Description
Perhaps sealer needs to provide an application image market to provide users with enough high-quality application images. For example, it provides directly available middleware capabilities such as database, message queue and configuration management. Further shielding the underlying complexity.

Type: *feature request*

**Comment by allencloud** [comment]
@lllwan Good idea! While sealer is open source project under CNCF. But I think that sealer applications marketplace seems to be a commercial product or at least operated by a commercial organization.

If it is a commercial one, I am afraid that maybe sealer open source team is not the suitable organization to do this. Actually I have little experience on this. Could you input some from the CNCF? @caniszczyk
